src/Esfa.Vacancy.Application/Commands/CreateApprenticeship/Validators/HoursPerWeekValidator.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Esfa.Vacancy.Application.Commands.CreateApprenticeship.Validators;
+using Esfa.Vacancy.Domain.Validation;
+using FluentValidation;
+
+namespace Esfa.Vacancy.Application.Commands.CreateApprenticeship
+{
+    public partial class CreateApprenticeshipRequestValidator
+    {
+        private const double HoursPerWeekMinimumLength = 16;
+        private const double HoursPerWeekMaximumLength = 48;
```

Comment:
48? Is there a way we can make this upper limit configurable. E.g. Reference data in a database. I am just thinking about the issues we have/had with WageRange